apps/system/test/marionette/edges_gesture_test.js

```diff
@@ -15,10 +15,6 @@ marionette('Edges gesture >', function() {
       'devtools.debugger.forbid-certified-apps': false
     },
     settings: {
-      'ftu.manifestURL': null,
-      'lockscreen.enabled': false,
-      'edgesgesture.debug': true,
-      'edgesgesture.enabled': true,
```

Comment:
I don't think we should as edgesgesture.enabled are now enabled everywhere by default. Even though this test is specific to edge gesturing, I don't see them getting turned off any time soon.